### ISSUE TITLE
Failures on go get github.com/testcontainers/testcontainers-go

### ISSUE THREAD

**Issue by pmalekn** [testcontainers/testcontainers-go#43]
When using `go1.11.4` and issuing `go get github.com/testcontainers/testcontainers-go` I get the following:

```
$ go get -v github.com/testcontainers/testcontainers-go
github.com/testcontainers/testcontainers-go
# github.com/testcontainers/testcontainers-go
/Users/USER/.gvm/pkgsets/go1.11.4/global/src/github.com/testcontainers/testcontainers-go/docker.go:116:32: cannot use inspect.NetworkSettings.NetworkSettingsBase.Ports (type "github.com/docker/docker/vendor/github.com/docker/go-connections/nat".PortMap) as type "github.com/docker/go-connections/nat".PortMap in return argument
/Users/USER/.gvm/pkgsets/go1.11.4/global/src/github.com/testcontainers/testcontainers-go/docker.go:197:25: multiple-value uuid.NewV4() in single-value context
/Users/USER/.gvm/pkgsets/go1.11.4/global/src/github.com/testcontainers/testcontainers-go/docker.go:219:3: cannot use exposedPortSet (type map["github.com/docker/go-connections/nat".Port]struct {}) as type "github.com/docker/docker/vendor/github.com/docker/go-connections/nat".PortSet in field value
/Users/USER/.gvm/pkgsets/go1.11.4/global/src/github.com/testcontainers/testcontainers-go/docker.go:261:3: cannot use exposedPortMap (type map["github.com/docker/go-connections/nat".Port][]"github.com/docker/go-connections/nat".PortBinding) as type "github.com/docker/docker/vendor/github.com/docker/go-connections/nat".PortMap in field value
```

**Comment by pmalekn** [comment]
So actually one of those errors is coming from API change from uuid package. It's easy to fix by adding a second variable in return value in [`docker.go` line 197](https://github.com/testcontainers/testcontainers-go/blob/master/docker.go#L197) and even better using https://github.com/gofrs/uuid instead of github.com/satori/go.uuid because it seems it's maintained anymore

```
    sessionID, err := uuid.NewV4()
    if err != nil {
      return nil, err
    }
```

**Comment by guenhter** [comment]
@pmalekn added your suggested fix to https://github.com/testcontainers/testcontainers-go/pull/44

The remaining problems seems to have their cause in docker's vendored dependencies (https://stackoverflow.com/questions/38091816/packages-type-cannot-be-used-as-the-vendored-packages-type) ...

**Comment by guenhter** [comment]
Have a look at
https://github.com/golang/go/wiki/Modules

It seems that modules are used in this project and when I activate that in my project too, everything works out.

**Comment by gianarb** [comment]
Yes, we use go modules, I have also released `v0.0.2` that should be the one that puts everything in order from our side.
If there are projects hat are using the old name we should fix them for atm this issue can be closed.
Thanks a lot and sorry for the BC break caused by the rename!
